TypeSystem/Schema/golden/smoke_test_struct.cc

```diff
@@ -69,7 +69,7 @@ struct TemplatedInheriting_T9200000002835747520 : A {
 };
 struct Templated_T9209626390174323094 {
   int32_t foo;
-  TemplatedInheriting_T9200000002835747520 bar;
+  TemplatedInheriting_Z bar;
```

Comment:
What if there are two or more templated with different template typenames? Which one of them will this reference?